ocs_ci/ocs/resources/cloud_manager.py

```diff
@@ -127,6 +127,28 @@ def get_all_uls_names(self):
     def verify_uls_exists(self, uls_name):
         pass
 
+    def verify_uls_state(self, uls_name, is_available):
+        check_type = "Delete"
+        if is_available:
+            check_type = "Create"
+        sample = TimeoutSampler(
+            timeout=180, sleep=15, func=self.verify_uls_exists, uls_name=uls_name
+        )
+        if sample.wait_for_func_status(result=is_available):
+            logger.info(
+                f"Underlying Storage {uls_name} {check_type.lower()}d successfully."
+            )
+        else:
+            if is_available:
+                raise ResourceInUnexpectedState(
+                    f"{check_type}ion of Underlying Storage {uls_name} timed out. "
```

Comment:
`{check_type}ion` yields `Createion` or `Deleteion` :)

Reply:
you are right fixed it :)